src/types/Typechecker/Typechecker.hs

```diff
@@ -32,6 +32,7 @@ import AST.Util(extend)
 import Types as Ty
 import Typechecker.Environment
 import Typechecker.TypeError
+import Typechecker.Backtrace
 import Typechecker.Util
 import Text.Printf (printf)
 import Debug.Trace
```

Comment:
this is probably our fault but, if we do not need the Trace module, could you remove it? Thanks